Week2task2/src/Name.java

```diff
@@ -0,0 +1,69 @@
+
+public class Name {
+	
+	String name;
+	String surname;
+	/**
+	 * @param name
+	 * @param surname
+	 */
+	public Name(String name, String surname) {
+		super();
+		this.name = name;
+		this.surname = surname;
+	}
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public String getSurname() {
+		return surname;
+	}
+
+	public void setSurname(String surname) {
+		this.surname = surname;
+	}
+
+	public Name() {
+		// TODO Auto-generated constructor stub
+	}
+	
+	public Name(Name person) {
+		name = person.name;
+		surname = person.surname;
+	}
+	
+	// check if one Name object is equal to another Name object
+	public boolean equal(Name person1) {
```

Comment:
The method should be called equals, because here we are overwriting the equals method from our parent, the Object class.